packages/tezos-protocol-environment-packer/tezos-protocol-environment-packer.8.0~rc2/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+maintainer: "contact@tezos.com"
+authors: [ "Tezos devteam" ]
+homepage: "https://www.tezos.com/"
+bug-reports: "https://gitlab.com/tezos/tezos/issues"
+dev-repo: "git+https://gitlab.com/tezos/tezos.git"
+license: "MIT"
+depends: [
+  "dune" { >= "2.0" }
+]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["mv" "src/lib_protocol_environment/%{name}%.install" "./"]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+synopsis: "Tezos: sigs/structs packer for economic protocol environment"
+
```

Comment:
This one needs to be ocaml-version >= 4.03 at least due to String.capitalize_ascii